src/front/app/features/challenge/challenge.actions.js

```diff
@@ -26,15 +27,18 @@ export function fetchChallenge(id) {
 
 export function watchChallenge(challengeId) {
     return dispatch => api(`/api/my/wish-list/${challengeId}`, { method: 'PUT' })
```

Comment:
@wips It is better to put to /api/my/wish-list with challengeId in body. So you will put new resource to wish-list, not to wish-list/{challengeId}
But, it is my opinion.